SpatialGDK/Extras/schema/actor_set_member.schema

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+package unreal;
+
+component ActorSetMember {
+	id = 9950;
+	
+	EntityId leader_entity = 1;
```

Comment:
Is it the only data necessary? I expect the set leader at least to have more information, like the number of set members.

Reply:
It seems to me that it is; do we need anything but `leader_entity` for the Strategy worker to be able to group actors?

Reply:
In terms of building the set, you are right. But in terms of receiving a set we do not have enough information to know when we have received all of it.

Reply:
I think my idea was to make the bare minimum. We'll certainly modify this component when we get to implementing the Strategy worker in earnest, but I'm not sure I'm ready to design the API at this point.

Reply:
Realized we could exploit this information even if it means we have implemented 50% of the actor set feature. Looks fine then.